### DISCUSSION TITLE
Disable update popup

### DISCUSSION THREAD

**Discussion by Jrmy003** [dbeaver/dbeaver#14464] [Q&A]
Hello,

I have disabled the automatic updates, but dbeaver keep asking for upgrades.

I'm still using the 21.0.1.

![image](https://user-images.githubusercontent.com/3603542/137759034-d8672718-173b-4a64-a6f4-633f94c55857.png)

**Comment by uslss** [comment]
@Jrmy003 ,
you should uncheck Automatic updates check option at User Interface tab in the Preferences
![image](https://user-images.githubusercontent.com/31996417/138051862-ffbd4d8c-5a38-49d5-a09e-d5553286fcc1.png)

**Comment by Jrmy003** [comment]
I try this, thank you 🙂
